NomadAudio/src/PreviewEngine.cpp

```
@@ -1,6 +1,7 @@
 // © 2025 Nomad Studios — All Rights Reserved. Licensed for personal & educational use only.
 #include "PreviewEngine.h"
 #include "NomadLog.h"
+#include "MiniAudioDecoder.h"
```

Cppcheck (2.18.0)

[information] Include file

(missingInclude)